baybe/simulation/wrappers.py

```diff
@@ -0,0 +1,243 @@
+"""Convenience wrappers for advanced simulations."""
```

Comment:
As said previously, I am not happy with the naming here in general, and also the term "advanced simulations" is not really to my liking. I, for instance, would call the transfer learning simulation a more advanced simulation than the one that we have here. I know that this is probably just temporary anyway, but I'd love if we could align on a different naming here already now :)

Reply:
hmm i must say i somehow agree, transfer learning wrapper is also a wrapper so this is a bit arb. Why not but all content off `wrappers` also in `core` ?
Any other partition we can come up with? Or triplet of `core` `scenarios` and `transfer_learning` ?

Reply:
I am very open to adjusting names (I'm also not super happy) but let's remember what is the main goal here: to split the stuff into easier digestible pieces and get away from the super monolithic file. It's really annoying to work with such long files and at some point we even got a linter error because of the enormous length. So would not want to put wrappers back into core.

* A triplet `core`, `something` and `transfer_learning` would be OK, but we still haven't agreed on the final function names and said that a renaming is needed, i.e. `simulate_experiment -> simulate_campaign` and `simulate_scenarios -> ... (yet unclear) ...`. Hence, perhaps not clever to use `scenarios` for the module.
* Could even split it further: `core`, `groupby`, ``<tbd> (for the current `simulate_scenarios` method`` and `transfer_learning`
* Any other choice you prefer ...

Reply:
for me keeping `simulate_scenarios` is fine, I think we discussed (and already decided) this some time ago as the only real alternative `simulate_campaignS` is too similar to the other one?

Reply:
For the sake of unblocking, maybe we should just agree on the naming **for now** and change this later? Also agree with keeping `simulate_scenarios`.

Reply:
Ok, decided then. Will keep `simulate_scenarios` for the function and it will be moved into a `scenarios` module, together with `_simulate_groupby`. Further, `simulate_experiments` will stay in `core` and later be renamed to `simulate_campaign` 👍🏼

Reply:
Done. Please verify if happy and resolve. PR is ready for merge from my end

Reply:
That is fine for me, will let @Scienfitz resolve if this also works for him.